dotnet/src/SemanticKernel.UnitTests/Planning/SKContextExtensionsTests.cs

```diff
@@ -42,7 +48,7 @@ public async Task CanCallGetAvailableFunctionsWithNoFunctionsAsync()
         var semanticQuery = "test";
 
         // Act
-        var result = await context.GetAvailableFunctionsAsync(config, semanticQuery);
+        var result = await context.GetAvailableFunctionsAsync(config, semanticQuery).ConfigureAwait(false);
```

Comment:
No need for the ConfigureAwait in the test.

Reply:
My IDE automatically does this so not going to worry about it right now.

Reply:
Actually I'll fix this too.

Reply:
#655